src/widgets/SearchModal/ui/SearchResultItem/SearchResultItem.tsx

```
@@ -0,0 +1,41 @@
+          <Badge size="sm" colorPalette="blue" variant="subtle">
+            {model.tableId}
+          </Badge>
+          <Text fontSize="xs" color="newGray.400">
+            Row: {model.rowId}
+          </Text>
+        </HStack>
+        <Box>
+          <Text fontSize="xs" color="newGray.400" mb={1}>
+            {model.path}
+          </Text>
+          <Text fontSize="sm" color="newGray.600" lineClamp={2} dangerouslySetInnerHTML={{ __html: model.highlight }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)